set up bug

with newest WebAPI from master, I get this at some point:


```
Caused by: org.flywaydb.core.internal.dbsupport.FlywaySqlScriptException:
Migration V2.2.5.20180215105415__separate-sequences.sql failed
--------------------------------------------------------------
SQL State  : 42P01
Error Code : 0
Message    : ERROR: relation "webapi.negative_controls_sequence" does not exist
  Position: 15
Location   : db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql (/usr/local/Cellar/tomcat/9.0.6/libexec/webapps/WebAPI/WEB-INF/classes/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql)
Line       : 14
Statement  : SELECT setval('webapi.negative_controls_sequence', coalesce(max(id), 1)) FROM webapi.concept_set_negative_controls
```

--- a/src/main/resources/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql
+++ b/src/main/resources/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql
@@ -4,15 +4,6 @@ SELECT setval('${ohdsiSchema}.cca_sequence', coalesce(max(cca_id), 1)) FROM ${oh
 CREATE SEQUENCE ${ohdsiSchema}.cohort_definition_sequence;
 SELECT setval('${ohdsiSchema}.cohort_definition_sequence', coalesce(max(id), 1)) FROM ${ohdsiSchema}.cohort_definition;
 
--- concept_set_sequence already exists - no need to create
-SELECT setval('${ohdsiSchema}.concept_set_sequence', coalesce(max(concept_set_id), 1)) FROM ${ohdsiSchema}.concept_set;
-
--- concept_set_item_sequence already exists - no need to create
-SELECT setval('${ohdsiSchema}.concept_set_item_sequence', coalesce(max(concept_set_item_id), 1)) FROM ${ohdsiSchema}.concept_set_item;
-
--- negative_controls_sequence already exists - no need to create
-SELECT setval('${ohdsiSchema}.CONCEPT_SET_NEGATIVE_CONTROLS_SEQUENCE', coalesce(max(id), 1)) FROM ${ohdsiSchema}.concept_set_negative_controls;
-
 CREATE SEQUENCE ${ohdsiSchema}.feasibility_study_sequence;
 SELECT setval('${ohdsiSchema}.feasibility_study_sequence', coalesce(max(id), 1)) FROM ${ohdsiSchema}.feasibility_study;
 

--- a/src/main/resources/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql
+++ b/src/main/resources/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql
@@ -12,3 +12,8 @@ SELECT setval('${ohdsiSchema}.ir_analysis_sequence', coalesce(max(id), 1)) FROM 
 
 CREATE SEQUENCE ${ohdsiSchema}.plp_sequence;
 SELECT setval('${ohdsiSchema}.plp_sequence', coalesce(max(plp_id), 1)) FROM ${ohdsiSchema}.plp;
+
+CREATE SEQUENCE ${ohdsiSchema}.negative_controls_sequence;
+SELECT setval('${ohdsiSchema}.negative_controls_sequence', coalesce(max(id), 1)) FROM ${ohdsiSchema}.concept_set_negative_controls;
+ALTER TABLE ${ohdsiSchema}.CONCEPT_SET_NEGATIVE_CONTROLS ALTER COLUMN id SET DEFAULT nextval('${ohdsiSchema}.negative_controls_sequence');
+DROP SEQUENCE IF EXISTS ${ohdsiSchema}.CONCEPT_SET_NEGATIVE_CONTROLS_SEQUENCE;

--- a/src/main/resources/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql
+++ b/src/main/resources/db/migration/postgresql/V2.2.5.20180215105415__separate-sequences.sql
@@ -11,7 +11,7 @@ SELECT setval('${ohdsiSchema}.concept_set_sequence', coalesce(max(concept_set_id
 SELECT setval('${ohdsiSchema}.concept_set_item_sequence', coalesce(max(concept_set_item_id), 1)) FROM ${ohdsiSchema}.concept_set_item;
 
 -- negative_controls_sequence already exists - no need to create
-SELECT setval('${ohdsiSchema}.negative_controls_sequence', coalesce(max(id), 1)) FROM ${ohdsiSchema}.concept_set_negative_controls;
+SELECT setval('${ohdsiSchema}.CONCEPT_SET_NEGATIVE_CONTROLS_SEQUENCE', coalesce(max(id), 1)) FROM ${ohdsiSchema}.concept_set_negative_controls;
 
 CREATE SEQUENCE ${ohdsiSchema}.feasibility_study_sequence;
 SELECT setval('${ohdsiSchema}.feasibility_study_sequence', coalesce(max(id), 1)) FROM ${ohdsiSchema}.feasibility_study;

--- a/src/main/resources/db/migration/postgresql/V2.3.0.20180330124512__add_source_permissions.sql
+++ b/src/main/resources/db/migration/postgresql/V2.3.0.20180330124512__add_source_permissions.sql
@@ -14,7 +14,7 @@ INSERT INTO ${ohdsiSchema}.sec_role_permission (role_id, permission_id)
         AND sr.name IN ('admin');
 
 INSERT INTO ${ohdsiSchema}.sec_permission(id, value, description) VALUES(nextval('${ohdsiSchema}.sec_permission_id_seq'), 'source:post', 'Create source');
-INSERT INTO ${ohdsiSchema}.sec_role_permission
+INSERT INTO ${ohdsiSchema}.sec_role_permission (role_id, permission_id)
   SELECT sr.id, sp.id
   FROM ${ohdsiSchema}.sec_permission sp,
     ${ohdsiSchema}.sec_role sr

